rest/blip_api_collections_test.go

```diff
@@ -28,6 +28,7 @@ func TestBlipGetCollections(t *testing.T) {
 						},
 					},
 				},
+				UseViews: base.BoolPtr(false),
```

Comment:
Given the current test structure (where the suite gets run with views or GSI), should these tests be doing a skip based on TestsDisableGSI(), like we do for the N1QL tests, instead of trying to force views=false?  I feel like this approach might break with a test bucket pool that's initializing for views.

Reply:
I would say yes in general, but as per thread with Marks, until we have GSI tests running at all, these are the only tests of collections. I'd probably do this when running tests with GSI becomes default under Jenkins.

Reply:
This is too confusing even if it happens to work, and will be moot once we move to testing GSI regularly with jenkins. In the meantime, someone working on collections should remember to run with `SG_TEST_USE_GSI=true go test -run "TestCollection"`